ocpmodels/trainers/forces_trainer.py

```diff
@@ -658,6 +660,14 @@ def _compute_metrics(self, out, batch_list, evaluator, metrics={}):
         metrics = evaluator.eval(out, target, prev_metrics=metrics)
         return metrics
 
+    def _check_traj_files(self, batch):
```

Comment:
Should we move this function to `ocpmodels/common/utils.py`?

Reply:
Sounds good. Will do.